lib/ASTGen/.swift-format

```diff
@@ -0,0 +1,17 @@
+{
+    "version": 1,
+    "lineLength": 120,
```

Comment:
FYI we settled on 160 [here](https://github.com/apple/swift/pull/66044#discussion_r1206085959), which motivated https://github.com/apple/swift-syntax/pull/1714.

How about a symlink to the swift-syntax configuration file for everything inside `lib` instead? Consistent formatting across our Swift repos sounds really nice.

Reply:
> but AFAIK we want to standardize on 120

Oh, that’s new. Totally fine by me! I still incline to a symlink until swift-syntax transitions though. I’m wondering what we can do to standardize the Swift formatting in these repositories without having to maintain multiple config files.

Reply:
Yes, we agreed to use 120 columns. We should update swift-syntax to 120 as well at some point but I haven’t gotten to it yet.

> I still incline to a symlink until swift-syntax transitions though

If we were to symlink swift-syntax’s config file to swift, it means that you can’t work on swift-syntax as a standalone package anymore. Similar for sourcekit-lsp. I think the best thing to do, is to have a separate config file in each repo and add a CI check that all the config files are the same.

Reply:
> If we were to symlink swift-syntax’s config file to swift, it means that you can’t work on swift-syntax as a standalone package anymore.

I meant the symlink to be in the swift repository.

> I think the best thing to do, is to have a separate config file in each repo and add a CI check that all the config files are the same.

Alternatively, maybe we could create a special repo to host this kind of stuff and add it as a git submodule to other repos. Then each repo can replace their own config files with symlinks to the config files in that submodule.